README.md

```diff
@@ -141,17 +141,18 @@ The STL uses boost-math headers to provide P0226R1 Mathematical Special Function
 acquire this dependency.
 
 1. Install Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+2. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL to this repository, typically
+   https://github.com/microsoft/STL.
+3. Change directories to the folder where this STL repository has been cloned.
+4. Invoke `cd vcpkg`. Alternatively, you can use a separate installation of vcpkg.
+5. Invoke `.\bootstrap-vcpkg.bat`
+6. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
    to install the boost-math dependency. Add `boost-math:arm-windows boost-math:arm64-windows` to this to target ARM
    and ARM64.
-6. Run `.\vcpkg.exe integrate install` which tells Visual Studio which vcpkg instance you wish to use. If you have never
+7. Run `.\vcpkg.exe integrate install` which tells Visual Studio which vcpkg instance you wish to use. If you have never
    done this before, you may be prompted to elevate.
-7. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL to this
-   repository, typically `https://github.com/microsoft/STL`
-8. Choose the architecture you wish to build in the IDE, and build as you would any other project. All necessary CMake
+8. Open Visual Studio, and choose the "Open a local folder" option. Navigate to the folder where this STL repository has been cloned.

```

Comment:
In my test I had to reopen the IDE after running "vcpkg.exe integrate install" in step 7 to make the build successfully.

Could we modify step 8 as below-

8. Reopen Visual Studio, and choose the "Open a local folder" option. Navigate to the folder where this STL repository has been cloned.

Reply:
Makes sense to me. Could also mention to close it after it's finished cloning. 

Though this is starting to seem like a pretty poor experience honestly. Having to wait for VS to clone the repo and also checkout `llvm_project` and `vcpkg` just to close it straight after just to reopen once `boost::math` is installed. 

@StephanTLavavej thoughts?

Reply:
I think the user could clone this repository, init vcpkg and install boost math using command line only and then could open the repository folder using IDE for build. 
This should have 2 benefits -
1. user would need to open the IDE only once when building. 
2. when using IDE to clone the repository the llvm_project submodule also inits which takes longer time although its not required during the build step.

Reply:
I agree that having to reopen the IDE seems complicated (and error-prone). If having to use the command line is unavoidable, then it probably makes sense to avoid any IDE reopening, even if that means executing more one-time command line instructions. However, I don't use the IDE in my workflow, so my opinion is not very valuable here.

Reply:
Well the reopening is only if they are going to use the vcpkg submodule. So if they have a vcpkg installation they were to use then they don't have to reopen.

Don't know how you would write instructions to incorporate all 3(?) scenarios without it getting unwieldy.

Reply:
So in case of build steps using IDE should we move ahead with using the command line for cloning the STL repository, init of vcpkg and boost math installation using command line and then using IDE for build?